src/statsmodels/statsmodel.jl

```diff
@@ -48,8 +48,8 @@ for (modeltype, dfmodeltype) in ((:StatisticalModel, DataFrameStatisticalModel),
                                  (:RegressionModel, DataFrameRegressionModel))
     @eval begin
         function StatsBase.fit{T<:$modeltype}(::Type{T}, f::Formula, df::AbstractDataFrame,
-                                              args...; kwargs...)
-            mf = ModelFrame(f, df)
+                                              args...; contrasts::Dict = Dict(), kwargs...)
```

Comment:
This would preclude using un-instantiated contrasts (e.g., `HelmertCoding` rather than `HelmertCoding()`) which is currently supported. If you think that's too clever we could restrict it like you suggest, but it's an idiom I borrowed from Gadfly.jl.